en/setup/dev_env_linux.md

```diff
@@ -1,10 +1,33 @@
-# Development Environment on Linux
+# Development Environment on Ubuntu LTS / Debian Linux
 
-We have standardized on Debian / Ubuntu LTS as the supported Linux distribution, but [boutique distribution instructions](../setup/dev_env_linux_boutique.md) are available for Cent OS and Arch Linux.
+> **Tip** We have standardized on Debian / Ubuntu LTS as the supported Linux distribution. Installation instructions are also provided for [boutique distributions](../setup/dev_env_linux_boutique.md) (Cent OS and Arch Linux).
+
+PX4 development on Linux supports four main families:
+
+* NuttX based hardware: [Pixhawk](../flight_controller/pixhawk.md), [Pixfalcon](../flight_controller/pixfalcon.md),
+  [Pixracer](../flight_controller/pixracer.md), [Pixhawk 3 Pro](../flight_controller/pixhawk3_pro.md), [Crazyflie](../flight_controller/crazyflie2.md),
+  [Intel® Aero Ready to Fly Drone](../flight_controller/intel_aero.md)
+* [Qualcomm Snapdragon Flight hardware](../flight_controller/snapdragon_flight.md)
+* Linux-based hardware: [Raspberry Pi 2/3](../flight_controller/raspberry_pi_navio2.md), Parrot Bebop
+* Simulation: [jMAVSim SITL](../simulation/sitl.md) and [Gazebo SITL](../simulation/gazebo.md)
+
+
+## Convenience Bash Scripts
```

Comment:
Attempt to simplify installation with bash scripts. Does this work? Are they the "right" bash scripts? Should we instead have without simulator versions as well, or for separate simulators. NOTE, I chose this set because I don't want to maintain many of these!